.ci/pipeline/release.yml

```diff
@@ -0,0 +1,221 @@
+#===============================================================================
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+variables:
+  DESCRIPTION: ReleaseTesting
+  DESELECT: --deselect ::test_svc_clone_with_callable_kernel --deselect ::test_precomputed
+  TEST_COMMAND: python -m sklearnex -m pytest -ra --disable-warnings --pyargs sklearn.svm.tests.test_svm $(DESELECT)
+
+jobs:
+- job: ReleasePyPi
+  strategy:
+    matrix:
+      ubuntu - python38:
```

Comment:
Suggest to add MaxParralel limit here - for example 5 to ensure that this testing will not consume all resources and will keep something for CI.  This should be done for each strategy block

https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#multi-job-configuration